Celeste.Mod.mm/Mod/Module/EverestModuleAssemblyContext.cs

```diff
@@ -520,14 +520,7 @@ public sealed class EverestModuleAssemblyContext : AssemblyLoadContext, IAssembl
                     // We have to unzip the native libs into the cache
                     string cachePath = Path.Combine(Everest.Loader.PathCache, "unmanaged-libs", UnmanagedLibraryFolder, ModuleMeta.Name);
 
-                    string modHash = (
-                        ModuleMeta.Hash
-                        ?? ModuleMeta.Multimeta.FirstOrDefault(meta => meta.Hash != null).Hash
-
-                        // if we're here, this means a P/Invoke occured in Load(); calculate the hash real quick
-                        // https://github.com/EverestAPI/Everest/issues/761
-                        ?? (ModuleMeta.Hash = Everest.GetChecksum(ModuleMeta))
-                    ).ToHexadecimalString();
+                    string modHash = (ModuleMeta.Hash ?? ModuleMeta.Multimeta.FirstOrDefault(meta => meta.Hash != null).Hash).ToHexadecimalString();
```

Comment:
Hmm, now that `ModuleMeta.Hash` can never be null, ig this could be cleaned up to just `ModuleMeta.Hash.ToHexadecimalString()`?

Reply:
That's true, I might as well do that.